packages/owl-bot/src/copy-code.ts

```diff
@@ -84,6 +88,8 @@ export async function copyCodeAndCreatePullRequest(
   const workDir = tmp.dirSync().name;
   logger.info(`Working in ${workDir}`);
 
+  // TODO(bcoe): is this clone necessary if we're already cloned the
```

Comment:
This was confusion on my part, we're cloning the repo being pushed to not the `googleapis-gen` repo.